hasweb/pages/about/index.md

```diff
@@ -2,20 +2,28 @@ title: About HasGeek
 
 HasGeek creates discussion spaces for geeks.
 
-We made
-[DocType HTML5](http://www.doctypehtml5.in/),
-[AndroidCamp](https://androidcamp.hasgeek.com/),
-[Scaling PHP in the Cloud](https://phpcloud.hasgeek.com/),
-[JSFoo](https://jsfoo.in/),
-[Droidcon India](https://droidcon.in/),
-the [Cartonama Workshop](https://workshop.cartonama.com/),
-[Meta Refresh](https://metarefresh.in/) and
-[Rootconf](https://rootconf.in/).
-We host an [IRC channel](irc://irc.freenode.net/#hasgeek)
-and a [job board](https://jobs.hasgeek.com/).
-We publish [our code](https://github.com/hasgeek).
-We are on [Twitter](https://twitter.com/hasgeek) and [Facebook](https://www.facebook.com/HasGeek).
-We occasionally host a [Geekup](https://geekup.in/) with visitors to our office in Bangalore.
-We conduct [hacknights](https://hacknight.in/) alongside our events. Come to our events to meet like-minded people, learn new things, and discover new opportunities.
-
-[See what else is coming up in the Funnel](https://funnel.hasgeek.com). [Sign up to be notified when we do an event](http://eepurl.com/duGmH).
+Previously, we've ran editions of:
+
+* [DocType HTML5](http://www.doctypehtml5.in/)
+* [AndroidCamp](https://androidcamp.hasgeek.com/)
+* [Scaling PHP in the Cloud](https://phpcloud.hasgeek.com/)
+* the [Cartonama Workshop](https://workshop.cartonama.com/)
+* [Geekup](https://geekup.in/)
+* [hacknights](https://hacknight.in/)
+
```

Comment:
Should I include that in previous events or currently running editions? because I don't remember zainab saying we will conduct again this year..